+++ src/Makefile.in	2025-05-11 23:04:00.117260722 +0200
@@ -407,6 +407,7 @@
 
 CFLAGS = @CFLAGS@
 LDFLAGS = @LDFLAGS@
+LDFLAGS += $(LDLIBS)
 LIBCFLAGS = $(CFLAGS)
 CXXFLAGS = @CXXFLAGS@
 LIBCXXFLAGS = $(CXXFLAGS) -fno-implicit-templates
@@ -2448,7 +2449,7 @@
 # Installation targets.
 
 .PHONY: install uninstall
-install:
+_install:
 	@: $(MAKE); $(unstage)
 	@r=`${PWD_COMMAND}`; export r; \
 	s=`cd $(srcdir); ${PWD_COMMAND}`; export s; \
@@ -50293,4 +50294,8 @@
 .NOEXPORT:
 MAKEOVERRIDES=
 
+GOA_TARGETS := bfd binutils gas ld intl opcodes
+goa: $(addprefix install-strip-,$(GOA_TARGETS))
+$(addprefix install-strip-,$(GOA_TARGETS)) : $(addprefix all-,$(GOA_TARGETS))
+
 # end of Makefile.in
