+++ src/Swipe.cpp	2023-07-05 16:02:21.238696126 +0200
@@ -3,8 +3,10 @@
 
 #include <stdio.h>
 
+#ifndef GENODE
 #include <X11/Xlib.h>
 #include <X11/Xatom.h>
+#endif /* GENODE */
 
 SDL_SysWMinfo
 Swipe::m_syswminfo = {0};
@@ -12,6 +14,7 @@
 void
 Swipe::lock(bool locked)
 {
+#ifndef GENODE
     unsigned int customRegion[] = {
         0,
         0,
@@ -36,5 +39,6 @@
             m_syswminfo.info.x11.wmwindow, customRegionAtom,
             XA_CARDINAL, 32, PropModeReplace,
             reinterpret_cast<unsigned char*>(&customRegion[0]), 4);
+#endif /* GENODE */
 }
 
